slurmci-test.toml

```diff
@@ -31,7 +31,6 @@ cpu_gpu = [
   { file = "test/LinearSolvers/poisson.jl", slurmargs = ["--ntasks=2"], args = [] },
   { file = "test/LinearSolvers/columnwiselu.jl", slurmargs = ["--ntasks=1"], args = []},
   { file = "test/LinearSolvers/bandedsystem.jl", slurmargs = ["--ntasks=3", "--time=02:00:00"], args = [] },
-  { file = "test/Ocean/shallow_water/GyreDriver.jl", slurmargs = ["--ntasks=1"], args = [] },
```

Comment:
I removed this since it isn't there anymore, but if it has just moved/been renamed, please re-add.

Reply:
ah I completely broke all the shallow water stuff, I'll fix it

Reply:
actually didn't break it, just moved things around.

I need to convert this to use the Driver + Config framework too, but I think I'll do that in a separate PR